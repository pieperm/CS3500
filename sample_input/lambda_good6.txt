(progn
  ((lambda (x) (print x)) "foo")
  ((lambda (x) (print x)) 1)
)
