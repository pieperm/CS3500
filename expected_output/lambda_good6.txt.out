
___Entering new scope...

___Adding x to symbol table

___Exiting scope...


___Entering new scope...

___Adding x to symbol table

___Exiting scope...

EXPR type is: INT_OR_STR_OR_BOOL
---- Completed parsing ----


Bye!
